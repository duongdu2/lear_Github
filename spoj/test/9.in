31
50
636 6977 854 6098 9629 4456 3409 1953 1954 4934 2141 4604 5796 8972 2453 1647 7082 8504 6755 2060 5035 295 4802 7178 741 4481 8440 4503 276 9434 6069 8163 2486 5998 9927 7448 3772 8757 542 7540 4378 6548 9209 1593 4945 7518 996 6376 2438 8312 
32
5471 8799 4057 5531 679 9562 562 5862 8856 5815 2165 2454 8595 4122 7186 4350 3273 609 7228 8428 3800 3805 950 1997 61 4321 6443 1285 5834 4236 5019 6726 
16
9444 8251 2291 2893 2710 7540 2417 7687 236 7481 9898 8666 343 4158 7700 4438 
49
9746 8735 5435 1509 1106 6674 1045 9066 3416 4456 3600 1960 1812 8302 6202 2363 4429 9830 8735 8924 8995 3199 3668 802 8786 1748 3254 3764 8783 2390 4794 2885 4889 271 1970 2469 2674 1665 1218 9763 521 4826 1380 1760 9744 8797 7962 6351 9049 
43
2368 3516 4433 4999 6127 430 906 5131 1453 4259 1209 185 3528 5759 5381 8740 9304 9893 5249 6770 4426 315 375 4882 9423 308 4236 797 3547 3394 3372 2420 4241 7311 1898 3134 8069 7628 5480 6990 8340 7943 8962 
26
4563 4514 3063 6084 4229 4717 1649 3914 3874 9419 2330 4771 8188 6437 1045 402 2516 6472 3449 7158 9167 9711 4114 9835 8299 8188 
32
9408 1359 9233 7435 3768 2757 5750 3418 3576 6312 6063 8118 8923 5353 1734 870 5721 1423 3462 7219 4563 5618 9436 1625 5565 8614 7385 7488 845 515 1574 8247 
8
4667 3701 1578 59 5583 3163 6114 7294 
26
4725 5780 1395 200 1096 2670 2479 4488 8482 3840 1168 7449 3029 3922 3273 1337 5909 5989 7784 5645 3579 5519 1675 6948 6057 5446 
14
8976 7542 2355 231 382 7070 4038 5547 4091 4297 5573 4785 4256 6345 
9
1397 5667 5782 9701 7893 8930 5756 6612 5530 
4
6781 3812 1753 8029 
46
9975 3431 7402 5130 2155 2948 3817 3305 3105 4236 6221 8316 2929 8137 4861 5946 6386 282 1985 3158 1881 8935 245 9007 1765 1625 3642 9335 8163 934 7169 281 9481 3120 1188 3798 2812 6724 5257 5298 7373 8465 368 3473 1952 1000 
46
8873 6908 6821 9696 8731 2527 3548 821 2401 715 1424 7010 7676 1291 3278 6263 9835 3385 4744 7950 6445 157 7943 6623 951 1421 1159 940 6824 6457 406 2219 5584 8724 2874 3541 4972 9263 7376 4390 418 769 4081 2253 5486 3776 
11
5199 9423 4309 8436 8615 2583 8555 9322 288 9672 1993 
34
8057 2627 4925 8922 9745 1410 6574 7081 5552 922 3586 5339 9568 3676 4700 9373 5104 9070 824 5470 5708 7382 6947 3276 1345 6035 3189 4833 1020 6438 2510 6238 3362 6015 
4
2011 8259 3696 6087 
20
1191 3984 3910 1532 6333 7775 5973 7723 8001 7003 6277 3720 1580 9200 239 1546 9961 7836 3610 2965 
50
1136 9628 5975 6764 5446 281 3530 9761 4467 6397 2546 519 2494 9535 2335 8232 1839 6832 6674 6506 4095 5073 5913 4359 9362 5674 1199 9680 101 6762 7990 4712 2782 9816 6078 3134 3611 418 3913 3447 3059 7992 7252 8872 3700 9967 4022 6099 1651 4101 
45
5492 4951 2841 5346 5890 2354 1924 5838 9740 1165 6473 1912 8958 1914 3328 7069 2848 7611 5895 4623 5435 7210 244 16 6040 6558 3946 9393 5838 7785 9624 7246 1076 9040 2728 6140 6399 1373 9017 7369 5964 8727 3945 6229 5461 
12
8209 5382 1182 5529 8178 4852 4622 3443 607 4756 5960 1100 
6
3865 1654 7590 6019 3136 2304 
42
4578 3868 1086 8108 1152 5621 8635 9913 9373 8147 3192 4817 4972 1362 1796 5941 2451 5908 5458 7220 6755 3051 5088 2087 3884 634 2930 2706 4395 2050 9031 3577 2073 5612 3111 6732 5370 4291 8827 4297 9408 2135 
32
5484 2307 670 3368 2491 8032 844 7131 3769 4348 8617 6545 8687 4396 3920 7247 1368 3122 753 7648 6897 1489 5090 7103 8232 235 2228 23 9252 4926 9406 1541 
41
5319 9856 6529 5861 1325 7661 4890 5651 421 9374 878 1541 8439 3188 5833 9248 8373 9778 4238 2289 220 4230 9074 5908 9093 5970 8668 4792 566 2643 4997 7813 6687 179 9534 8277 7382 4038 7370 3743 8646 
31
5232 8717 3824 1400 2217 553 4823 6866 911 9125 7695 391 3837 869 400 2537 2603 1887 166 6267 5334 7285 7278 7137 7765 4550 221 4478 6479 8837 427 
50
3710 7438 5589 1192 8774 7148 6647 6653 1513 4755 9882 2023 4689 7056 4134 22 6370 4410 2337 5210 5220 4788 2339 1972 4379 8295 4701 9734 5614 3184 5904 4464 7971 1523 2427 3522 6792 9023 6103 1952 1177 5331 6111 4242 1268 4995 3179 8793 9918 7052 
12
3619 3485 3662 8034 4992 8734 6733 5745 4211 7158 4897 1166 
15
5960 4248 4061 2704 3621 3541 15 9198 23 1256 8971 3584 6576 9711 3507 
9
9454 3469 8038 6078 9378 1758 3646 599 2571 
4
1976 1870 4156 7469 
