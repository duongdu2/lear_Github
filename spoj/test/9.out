28990
27217
28008
29339
28620
28965
28077
18991
23715
23588
26524
18622
28791
28794
28417
28686
18042
27162
29544
28757
22347
17474
28694
27345
29168
26679
29534
23926
27880
26870
13601
